### PR TITLE
Contact Info Ux Updates

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #: #5025

*Description of changes:*
* Decreased subtitle text size.
* Added No fee tool tip.
* Made Phone required.
* Added vertical action divider.

### CODE REVIEW

File: src/components/ContactInfo/ContactInfo.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
       <!-- Email Address -->
       <v-flex xs4>
         <label class="d-flex flex-wrap">
-          <span class="subtitle mr-2">Email Address</span>
+          <span class="subtitle text-body-3 mr-2">Email Address</span>
```
Comment (andrepestana-aot) [inline src/components/ContactInfo/ContactInfo.vue:19]: text-body-3 is set to
````
  .text-body-3 {
    font-family: inherit;
    font-size: 0.875rem !important;
    line-height: 1.375rem !important;
  }

but in the design doc it has 14px. What font-size should we use?
````

**Review by andrepestana-aot — APPROVED** [review]
Looks good to me!

File: src/components/ContactInfo/ContactInfo.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/ContactInfo/ContactInfo.vue:19]: 14px converted to rem is .875.

As a rule of thumb, we've traditionally used rem for font-sizes.

Comment (severinbeauvais) [reply src/components/ContactInfo/ContactInfo.vue:19]: https://www.google.com/search?q=why+use+rem+instead+of+px

**Review by jordiwes — APPROVED** [review]

**Review by severinbeauvais — APPROVED** [review]

File: src/components/ContactInfo/ContactInfo.vue (hunk 1 above)
Comment (andrepestana-aot) [reply src/components/ContactInfo/ContactInfo.vue:19]: 👍

**Pull request merged**